### PR TITLE
updated projects section

### PR DESCRIPTION
**Pull request by arfazca**
- [x] Hide hackathon section, replacing it with Projects
- [x] Populated Projects data with most recent and active projects
- [ ] Update ```Project Meeting Times``` with up-to-date data
- [ ] Add VikeLabs Transparent Logo beside Projects Header

### CODE REVIEW

File: pages/projects.tsx [hunk 1]
```diff
@@ -1,43 +1,184 @@
...
+      status: "Active",
+      tech: ["React", "Node.js", "MongoDB"],
+      meetingTime: "Undecided",
+      lastUpdate: "Fall 2023",
+      term: "Fall 2023"
+    }
+  },
+  {
+    id: 4,
+    fields: { slug: "/lockers" },
```
Comment (liamshatzel) [inline pages/projects.tsx]: Could we call this locker-registration? I think their full title is ESS Locker Registration

**Review by liamshatzel — CHANGES_REQUESTED** [review]
Looks great! Just one small change

File: pages/projects.tsx (hunk 1 above)
Comment (arfazca) [reply pages/projects.tsx]: @liamshatzel I just updated it, let me know if there's anything else!

**Review by liamshatzel — APPROVED** [review]
Looks good! Feel free to merge

**Pull request merged**